internal/server/v2/observation/golden/calculation/statistical_calculation.json

```diff
@@ -38,209 +38,209 @@
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-03",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-08",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-09",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-10",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-11",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-12",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-14",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-15",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-08-16",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-09-14",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-09-15",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-09-16",
-                    "value": "1"
-                  }
-                ],
-                "obs_count": "26",
-                "earliest_date": "2022-06-10",
-                "latest_date": "2022-09-16"
-              },
-              {
-                "facet_id": "263855571",
-                "observations": [
-                  {
-                    "date": "2022-06",
-                    "value": "2"
-                  },
-                  {
-                    "date": "2022-07",
-                    "value": "2"
-                  },
-                  {
-                    "date": "2022-08",
-                    "value": "1"
-                  },
-                  {
-                    "date": "2022-09",
-                    "value": "1"
-                  }
-                ],
-                "obs_count": "4",
-                "earliest_date": "2022-06",
-                "latest_date": "2022-09"
-              },
-              {
-                "facet_id": "341794991",
-                "observations": [
-                  {
-                    "date": "2022",
-                    "value": "5"
-                  }
-                ],
-                "obs_count": "1",
-                "earliest_date": "2022",
-                "latest_date": "2022"
-              }
-            ]
-          }
+          "ordered_facets": [
+            {
+              "observations": [
+                {
+                  "date": "1975",
+                  "value": 128821
+                },
```

Comment:
@n-h-diaz any idea why the diffs here?

Reply:
I'm not sure - the data looks the same, so maybe it fixed the formatting?